core/Plugin/Controller.php

```diff
@@ -632,7 +632,7 @@ protected function setGeneralVariablesViewAs($view, $viewType)
         $rawDate = Piwik::getDate();
         Period::checkDateFormat($rawDate);
 
-        $periodStr = Piwik::getPeriod();
+        $periodStr = Piwik::getPeriod('day');
```

Comment:
simply using `day` as default might make it a bit better I guess, but in theory it's possible to disable the `day` period for UI, causing an exception if it's used nevertheless. I guess it could make more sense to use the users default period / systems default period instead.

Also this will change the default period used for all twig templates. I think this shouldn't cause any regressions, but hard to say, as it actually affects all parts of Matomo and any plugin.